client/app/domain/domain-tabs.controller.js

```diff
@@ -131,10 +131,7 @@ angular.module('App').controller(
     }
 
     static getUpdateOwnerUrl(domain) {
-      if (
-        _.get(domain, 'whoisOwner', false)
-        && domain.whoisOwner !== 'pending'
-      ) {
+      if (_.isObject(domain.whoisOwner) && _.has(domain, 'whoisOwner.id')) {
```

Comment:
Isn't `has` enough in the end ?

Reply:
Sorry for the confusion. This change has been reverted